runtime-config.md

```diff
@@ -12,30 +12,7 @@ Only [mounts from the portable config](config.md#mount-points) will be mounted.
 
 *Example (Linux)*
 
-```json
-"mounts": {
-    "proc": {
-        "type": "proc",
-        "source": "proc",
-        "options": []
-    },
-    "dev": {
-        "type": "tmpfs",
-        "source": "tmpfs",
-        "options": ["nosuid","strictatime","mode=755","size=65536k"]
-    },
-    "devpts": {
-        "type": "devpts",
-        "source": "devpts",
-        "options": ["nosuid","noexec","newinstance","ptmxmode=0666","mode=0620","gid=5"]
-    },
-    "data": {
-        "type": "bind",
-        "source": "/volumes/testing",
-        "options": ["rbind","rw"]
-    }
-}
-```
+See Mounts [example](runtime-config-linux.md#mounts-in-linux) in Linux
 
 *Example (Windows)*
```

Comment:
Is it worth pulling the Windows (and hypervisor?  See #133) examples out into platform-specific files as well.  `runtime-config-windows.md`, etc., would match our current Linux pattern, but that would currently be a fairly sparse file.  Still, I expect it will grow in time, and breaking it out now will set the stage for cleaner future expansion.